proto/umee/leverage/v1/query.proto

```diff
@@ -79,7 +79,9 @@ message QueryParamsResponse {
 
 // QueryRegisteredTokens defines the request structure for the RegisteredTokens
 // gRPC service handler.
-message QueryRegisteredTokens {}
+message QueryRegisteredTokens {
+  string base_denom = 1;
+}
```

Comment:
Could you add an entry in `CHANGELOG.md` for this since it's an addition to the API?